src/.vitepress/components/community/partners/partner-item.vue

```diff
@@ -1,81 +1,19 @@
-<template>
-  <div class="partner">
-    <div class="logo">
-      <a :href="partner.urlLink">
-        <img :alt="`${partner.name}'s logo`" :src="logo" width="125" height="auto">
-      </a>
-    </div>
-    <div class="profile">
-      <p class="description">{{ partner.description }}</p>
-      <dl>
-        <dt>
-          <i class="fa fa-link"></i>
-          <span class="sr-only">Link</span>
-        </dt>
-        <dd>
-          <a :href="partner.urlLink">{{ partner.urlText }}</a>
-        </dd>
-
-        <dt>
-          <i class="fa fa-map-marker"></i>
-          <span class="sr-only">Location</span>
-        </dt>
-        <dd>{{ partner.location }}</dd>
-
-        <dt>
-          <i class="fa fa-globe"></i>
-          <span class="sr-only">Languages</span>
-        </dt>
-        <dd class="language-list">
-          <ul>
-            <li v-for="language in languages" :key="language">{{ language }}</li>
-          </ul>
-        </dd>
-
-        <dt title="Proficiencies">
-          <i class="fa fa-briefcase"></i>
-          <span class="sr-only">Proficiencies</span>
-        </dt>
-        <dd class="proficiency-list">
-          <ul>
-            <li v-for="proficiency in partner.proficiencies" :key="proficiency.name">
-              <a v-if="proficiency.url" :href="proficiency.url">
-                {{ proficiency.name }}
-              </a>
-              <template v-else>{{ proficiency.name }}</template>
-            </li>
-          </ul>
-        </dd>
-
-        <dd class="social">
-          <SocialIcon v-if="partner.email" type="Email" :link="mailHref"/>
-          <SocialIcon v-for="link in partner.socialLinks" :key="link.name" :type="link.name" :link="link.url"/>
-        </dd>
-      </dl>
-    </div>
-  </div>
-</template>
-
 <script>
 export default {
   props: {
     partner: Object
   },
 
-  components: {
-    SocialIcon: () => import('@theme/components/ui/SocialIcon.vue')
```

Comment:
I'm having a bit of issue importing the local `@theme` directory components for reuse within this page. Any pointers on this would be helpful. Sending more context in Discord